AirportTicketBookingSystem.csproj

```diff
@@ -5,6 +5,19 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+	 <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="AutoFixture" Version="4.18.1" />
```

Comment:
why these packages needed in the main project? I see the tests project existing twice in the solution:

![image](https://github.com/user-attachments/assets/3ffec4e7-7e5c-4126-ab5b-82155f6d8050)

We should remove the folder under `AirportTicketBookingSystem` project. You can fix this by following this folder structure:

![image](https://github.com/user-attachments/assets/42c81b81-f98f-49ff-9e9c-9b58f2754694)

Make sure to update the folders as above, and update `.sln` file to have the new path for the tests project:
`AirportTicketBookingSystem\AirportTicketBookingSystem.csproj"`